### PR TITLE
Add Schema Migrator to dev images build GA

### PR DESCRIPTION
**Pull request by szwedm**
**Description**

Build Schema Migrator image in PRs.

**Related issue(s)**

#1185

### CODE REVIEW

File: .github/workflows/pull-build-images.yaml [hunk 1]
```diff
@@ -80,4 +80,12 @@ jobs:
...
\ No newline at end of file
+         build-args: BIN=globalaccounts
+
+   schema-migrator-image:
+      uses: kyma-project/test-infra/.github/workflows/image-builder.yml@main
+      with:
+         name: kyma-environment-broker-schema-migrator
+         dockerfile: Dockerfile.schemamigrator
+         context: .
+         build-args: BIN=schema-migrator
```
Comment (MarekMichali) [inline .github/workflows/pull-build-images.yaml:91]: The schema migrator Dockerfile doesn't use build-args, so this section is not needed.

Comment (MarekMichali) [reply .github/workflows/pull-build-images.yaml:91]: Also don't use full `kyma-environment-broker-schema-migrator` name. Look how other jobs are named.

Comment (szwedm) [reply .github/workflows/pull-build-images.yaml]: I removed `build-args` input.

**Review by MarekMichali — APPROVED** [review]

**Pull request merged**